src/AppBundle/Form/MembershipChooseNearbyCommitteeType.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace AppBundle\Form;
+
+use AppBundle\Entity\Committee;
+use Doctrine\ORM\EntityRepository;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class MembershipChooseNearbyCommitteeType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('committees', EntityType::class, [
```

Comment:
It's a good idea but I'm unable to theme only the EntityType widget in this case. The hidden token field also pass in the theme block and get a label etc. The block_prefix / block_name have no effect.